pkg/cmd/roachtest/cluster.go

```diff
@@ -2434,7 +2434,11 @@ func (c *clusterImpl) RunE(ctx context.Context, options install.RunOptions, args
 			logFileName = l.File.Name()
 		}
 		createFailedFile(logFileName)
-		return errors.Wrapf(err, "full command output in %s.log", logFile)
+		if c.l.File != nil {
+			return errors.Wrapf(err, "full command output in %s.log", logFile)
```

Comment:
This looks great, but we should make the same change above around line 2420 too.

Reply:
Updated